packages/api/src/routes/pins-add.js

```diff
@@ -51,6 +51,12 @@ export async function pinsAdd(event, ctx) {
   })
 
   const upload = await db.createUpload({
+    pins: [
+      {
+        status: 'PinQueued',
+        service: 'ElasticIpfs', // via pickup
+      },
+    ],
```

Comment:
We do this to override the default pin info that `createUpload` would set if we didn't provide this. New pins via pickup are provided via E-IPFS.